### ISSUE TITLE
Check failed: ShapeEquals(proto) shape mismatch (reshape not set) ..?

### ISSUE THREAD

**Issue by JoshuaBatty** [pannous/caffe-speech-recognition#8]
Hey pannous, I managed to train the model over night and now trying to run the test-speech-recognition.ipynb. 

For some reason i'm getting the following 

F0722 12:16:15.248998 2071900928 blob.cpp:455] Check failed: ShapeEquals(proto) shape mismatch (reshape not set)
*** Check failure stack trace: *** 

Have you come across this before? Or know what might be causing that error in the script? 

Also I had to change the following line of code from:
net = caffe.Net(model,weights)

to: 
net = caffe.Net(model,weights,caffe.TEST)

Thanks.

**Comment by pannous** [comment]
Looks like coffee lost the ability to read monochrome color images:
the is_color: false flag was removed

Investigating the options ...

**Comment by JoshuaBatty** [comment]
Ok great, good to know it wasn't some error on my end. Looking forward to any potential solutions....

**Comment by JoshuaBatty** [comment]
Hi Pannous, if we used RGB Spectrograms like in the image below would that be a possible work around? 

![sxfft1](https://cloud.githubusercontent.com/assets/1289413/9296038/b6f81432-44c5-11e5-9da8-afb0f655f762.jpg)

**Comment by pannous** [comment]
Hi Joshua,

 you probably solved this yourself. If not:  yes in the pipeline could easily be reconfigured to consume colored spectrographs. Do you have a data dump available or was that just an example image?

**Comment by sunshinehuo** [comment]
Hi JoshuaBatty! I have met the same error with yours,it shows like:
I0416 19:15:01.163847 31334 net.cpp:235] This network produces output prob
I0416 19:15:01.164002 31334 net.cpp:482] Collecting Learning Rate and Weight Decay.
I0416 19:15:01.164043 31334 net.cpp:247] Network initialization done.
I0416 19:15:01.164062 31334 net.cpp:248] Memory required for data: 545512320
F0416 19:15:01.288144 31334 blob.cpp:455] Check failed: ShapeEquals(proto) shape mismatch (reshape not set)
*** Check failure stack trace: ***

How did you solve it at last?Do you have any ideas?Thanks very much!
